src/Test/FixturesTrait.php

```diff
@@ -159,4 +158,20 @@ protected function tearDown(): void
 
         parent::tearDown();
     }
+
+    /**
+     * @see KernelTestCase::createKernel()
+     */
+    private function determineEnvironment()
+    {
+        if (isset($_ENV['APP_ENV'])) {
```

Comment:
Is it possible that this value is never defined?

Reply:
As example, phpunit.xml.dist would allow you to set both the server and env parameters:

```
<env name="APP_ENV" value="test" />
<server name="APP_ENV" value="test" />
```

Not sure, but I even have the idea that `env` has precedence over `server`. You could define a different environment variable in your `docker-compose` config, and let tests run on the environment you expect by setting env in `phpunit.xml.dist`.